proxy/service.py

```
@@ -1217,3 +1280,21 @@ def set_cancel_queued_flow_run(flow_run_id: str, payload: CancelQueuedManualJob)
+        # Run the pm2 list command and capture the output
+        result = subprocess.run(["pm2", "list"], stdout=subprocess.PIPE, text=True)
+        # Filter the output to count the number of processes with the specified name
+        count = sum(
+            1
+            for line in result.stdout.splitlines()
+            if any(queue_name in line for queue_name in payload.work_queue_names)
+        )
+        return count
+    except Exception as err:
+        logger.exception(err)
+        raise PrefectException(f"failed to fetch workers: {e}") from e
```

Ruff (0.8.2)

Undefined name `e`

(F821)